### ISSUE TITLE
Examples and wording hard to follow

### ISSUE THREAD

**Issue by AdamBebko** [dotnet/docs#36052]
### Type of issue

Other (describe below)

### Description

The content of this page is super helpful, but I find the examples difficult to understand, especially because of the names of some variables (i.e. `ctr`). Also the wording of some of the explanations is extremely difficult to follow.

### Page URL

https://learn.microsoft.com/en-us/dotnet/standard/parallel-programming/chaining-tasks-by-using-continuation-tasks

### Content source URL

https://github.com/dotnet/docs/blob/live/docs/standard/parallel-programming/chaining-tasks-by-using-continuation-tasks.md

### Document Version Independent Id

a5fa3e78-73de-3360-1253-00545a2b4ef2

### Article author

adegeo

### Metadata

* ID: be073b14-6d0b-60be-7dff-4674bcb6c674
* Service: **dotnet-fundamentals**

---
[Associated WorkItem - 633651](https://dev.azure.com/msft-skilling/Content/_workitems/edit/633651)

**Comment by gewarren** [comment]
cc @adegeo
